modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/constants/pageTypes.js

```diff
@@ -19,6 +19,7 @@ const productsFormatted = Object.entries(
 ]);
 
 export const PAGE_TYPES = {
+	dxp_new: 'dxp_new',
```

Comment:
Properties name should be named as camelCase.

Be aware to change in all files that use it. :)

```suggestion
	dxpNew: 'dxp_new',
```

Reply:
If I recall correctly changing this will need to fix the getCurrentProduct function @joanagrego. Please take a good look there

Reply:
Yeah, we're using return a `join('_');`.
We have a file that handles the camelCase pattern, the getCamelCase function (https://github.com/pt-liferay-solutions/liferay-portal/blob/master/modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/getCamelCase.js)

Reply:
@iagochaves I think that we don't need to change this function because we're just changing the prop and not the string 'dxp_new'.